packages/core/directives/map.ts

```diff
@@ -241,14 +246,22 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
   /**
    * The map mapTypeId. Defaults to 'roadmap'.
    */
-  @Input() mapTypeId: 'roadmap'|'hybrid'|'satellite'|'terrain'|string = 'roadmap';
+  @Input() mapTypeId: 'roadmap' | 'hybrid' | 'satellite' | 'terrain' | string = 'roadmap';
 
   /**
    * When false, map icons are not clickable. A map icon represents a point of interest,
    * also known as a POI. By default map icons are clickable.
    */
   @Input() clickableIcons: boolean = true;
 
+  /**
+   * A map icon represents a point of interest, also known as a POI.
+   * When map icons are clickable by default, an info window is displayed.
+   * When this property is set to npm run buildfalse, the info window will not be shown but the click event
+   * will still fire
+   */
+  @Input() isToShowDefaultInfoWindowForIcons: boolean = true;
```

Comment:
The input name is rather verbose. A shorter name, such as `showDefaultInfoWindow`, would be easier to understand and would avoid cluttering user code down the line.

Reply:
@benjcallaghan i have committed fixes to all of your comments.
Thanks